examples/nextjs-spa/styles/globals.css

```diff
@@ -59,6 +59,10 @@ a {
   text-decoration: underline !important;
 }
 
+.contentContainer {
+  margin-top: 20vh;
```

Comment:
I don't like using `vh` for padding.
We can use rem or px instead since the nav bar is a set height always, regardless of the viewport size.